array-manipulation-1/src/reverse.js

```diff
@@ -0,0 +1,17 @@
+/* exported reverse */
+// PSEUDOCODE: Define a function named reverse that takes in a single parameter: array.
+// Define a variable named arr and assign it the value of an empty array.
+// Create a for loop with an initial statement of a variable i being assigned the value of the length of the array parameter subtracted by 1.
+// Give the for loop a conditional statement of i being greater than or equal to 0 so the loop knows to end when it reaches the first element of the array.
+// Give the for loop a final statement of post-decrementing the i variable.
```

Comment:
You could simply write someting like `create a loop that iterates over the entire array, in reverse order`

It'll be up to the developer reading the pseudocode to implement how that loop will work :)